.circleci/config.yml

```diff
@@ -6214,13 +6214,6 @@ workflows:
               only: postnightly
 
   docker_build:
-    triggers:
```

Comment:
this was removed for testing?

Reply:
Yes, it's needed to trigger the docker rebuild.